core/src/main/java/jenkins/views/Header.java

```diff
@@ -0,0 +1,13 @@
+package jenkins.views;
+
+import hudson.ExtensionPoint;
+
+public interface Header extends ExtensionPoint {
+    
+    /**
+     * Checks if header is enabled. By default it is if installed, but the logic is deferred in the plugins.
```

Comment:
Wrong comment. There is no default behavior

Reply:
You are right. Updated in https://github.com/jenkinsci/jenkins/pull/5909/commits/88eac160e4ec72ee1a60680d1a8f36da8c79242e